app/views/idv/inherited_proofing/retrieval.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('inherited_proofing.headings.retrieval') %>
+
+<%= render PageHeadingComponent.new.with_content(t('inherited_proofing.headings.retrieval')) %>
+
+<div class="shield-spinner margin-y-4" id='spinner'>
+  <div class='text-center'>
+    <%= image_tag(
+          asset_url('shield-spinner.gif'),
```

Comment:
We already have a spinner GIF at `assets/images/spinner.gif`. Can we use that instead of adding another copy of it?

Reply:
Oh, I see this is a different image (with a picture of the Login.gov logo instead of a lock icon). I wonder why it's different 🤔

Reply:
Ok I got some clarification that it's intended to be different 👍